adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -19,7 +19,7 @@ package org.bdgenomics.adam.rdd.read
 
 import java.io.StringWriter
 
-import htsjdk.samtools.{ SAMFileHeader, SAMTextHeaderCodec, SAMTextWriter, ValidationStringency }
+import htsjdk.samtools.{ TextCigarCodec, SAMFileHeader, SAMTextHeaderCodec, ValidationStringency, SAMTextWriter }
```

Comment:
Small nit: at this point, might it not make sense to just do `import htsjdk.samtools._`? I think the general guideline we've been using is to import the whole package if we are using more than 4 items from the package.